starter_project/normalize_breton_test.py

```diff
@@ -1,15 +1,85 @@
+"""Tests for evaluating Celtic initial consonant mutations."""
+
 import unittest
 import normalize_breton_lib
 
 class TestStringMethods(unittest.TestCase):
+    """Tests for evaluating initial consonant mutations."""
 
     def test_normalize_breton(self):
-        'Test the output of NormalizeBreton.'
-        test_cases = [(('a--bc', 'a-bc'), ('ccb--a', 'ccb-a'), ('ba--aa', 'ba-aa'))]
-        for test in test_cases:
-          for test_case, expected in test:
-            test_fst = normalize_breton_lib.NormalizeBreton(test_case)
+        'Test the output of normalize_breton.'
+        with open("testdata/bre_normalized_sentences.tsv", "r") as test_file:
+            test_cases = test_file.readlines()[1:]
+#        test_cases = [(("ma gwin Da gwin tri pesk bara kozh ha KIG",
+#                        "ma gwin da win tri fesk bara kozh ha kig"),
+#                       ("da Kemper", "da gemper"))]
+        for sentence in test_cases:
+            test_case = sentence.strip().split("\t")[1]
+            expected = sentence.strip().split("\t")[2]
+            test_fst = normalize_breton_lib.normalize_breton(test_case)
             self.assertEqual(test_fst, expected)
 
+
+    def test_normalize_breton_soft_mutation(self):
+        'Test the Breton soft mutation.'
+        test_cases = [(("Div plac'h", "div blac'h"),
```

Comment:
Consider using parameterized tests instead of this style in the main project (this is ok for the starter project).

Reply:
Done! I also got to learn a bunch about parameterizing tests and such, which will definitely be useful for the main project.